tests/test_plot.py

```diff
@@ -315,6 +315,18 @@ def test_2d_plots(function, handsome_signal_2d):
                      file_type, compare_output)
 
 
+@pytest.mark.parametrize('function', [
+    (plot.time_2d), (plot.freq_2d), (plot.phase_2d), (plot.group_delay_2d),
+    (plot.time_freq_2d), (plot.freq_phase_2d), (plot.freq_group_delay_2d)])
+def test_2d_plots_complex(function, handsome_signal_complex_2d):
+    """Test all 2d plots with default arguments."""
+    filename = function.__name__
```

Comment:
```suggestion
    filename = f'complex_2d_{function.__name__}'
```
I think otherwise it will cmp vs the wrong files.